app/models/like.rb

```diff
@@ -6,6 +6,6 @@ class Like < ApplicationRecord
   after_save :update_like_counter
 
   def update_like_counter
-    post.update(like_counter: post.like.count)
+    post.update(like_counter: post.likes.count)
```

Comment:
- [ ] **Fatuma** you did great on you project 👏. Add a post and a comment is working correctly. However, there is an issue white liking a post. Everytime we like a post the post counter get incremented with is not the speteded behaviour. So please ensure to fix that for a better user experience. 👌

### Issue demo :point_down: 
[Screencast from 10.11.2023 21:33:55.webm](https://github.com/fatmahussein/Blog-app/assets/89228743/3bf41a63-fd3f-4e9a-a853-7229ce6aa8d2)


After some debugging I have found that in the `Like` model in the `update_like_counter` method instead of updating the `likes`  `count`  on the post just by increament the `like_counter` will fix the issue. Like below :point_down: 


### Update :point_down: 
```suggestion
    post.increment!(:like_counter)
```

Reply:
@ridwanediallo Thank you. you are a gem !!